diff --git a/assets/update_config b/assets/update_config
index e44921d6..33f65ed4 100644
--- a/assets/update_config
+++ b/assets/update_config
@@ -6,7 +6,7 @@
 # * You can define any tags that end with 'Url'. These tags will be added in the UI automatically
 # * Chances are you only need to change the ota_url in this file 
 #
-ota_url=https://raw.githubusercontent.com/crdroidandroid/android_vendor_crDroidOTA/9.0/update.xml
+ota_url=https://raw.githubusercontent.com/killerdroid96/android_vendor_crDroidOTA/9.0/update.xml
 device_name=ro.lineage.device
  
 #
